book/website/community-handbook/onboarding-offboarding/offboarding-core-team.md

```diff
@@ -0,0 +1,53 @@
+:::
+
+# TTW Core Team Onboarding checklists
+
+## Accounts and Permissions
+- [ ] Remove from #ttw-core-team channel on Slack
+- [ ] Change [TTW Github repository](https://github.com/alan-turing-institute/the-turing-way) permissions as needed
+- [ ] Add to 'Alumni' section of [Ways of Working](https://github.com/alan-turing-institute/the-turing-way/blob/main/ways_of_working.md) page on TTW Github repository 
+- [ ] Remove from [Core Team Google group](https://groups.google.com/u/1/g/core-team-2022) (currently dormant)
+- [ ] (If applicable) Remove from HackMD team for The Turing Way
+- [ ] (If applicable) Give Accounts/Password Access on Keybase
+- [ ] (If applicable) Remove from [The Turing Way Core Google Drive](https://drive.google.com/drive/u/0/folders/1bdjA-fBotMi9CnjDqu84d5hPZWwjgi-x)
+    - [ ] Remove from [promotion pack](https://drive.google.com/drive/u/1/folders/1mzGmbJkPnP5q1goQesxDc_E5zAPL0eTF) folder
+    - [ ] Remove from [conference talks/workshop applications](https://drive.google.com/drive/u/0/folders/1eNHq9sXq6Z8Q5Ep0MzNS0GsB3FKgTi3E) folder
+- [ ] (If applicable) Remove from Canva team
+- [ ] (If applicable) Remove from Penpot team
+- [ ] (If applicable) Remove from Keybase group
+
+## Personal information
+- [ ] Move information on [TTW Team slides](https://docs.google.com/presentation/d/1PeQ_gg5tOJVhOA4aSnVumsowD-bfRFEroWfGsxxaGCw/edit#slide=id.g1aee8e18598_0_0) to 'Alumni' section
+- [ ] (If applicable) remove [TTW project page](https://www.turing.ac.uk/research/research-projects/turing-way) on Alan Turing website
+- [ ] (If applicable) remove yourself to [TPS/Whitaker Lab slide deck](https://docs.google.com/presentation/d/1g-plncmi00FK97QuYDfqq7oPzUmL0xkHm1hy4X81IX8/edit#slide=id.gf5e62e1b31_1_0)
+
+## Events & Calendars
+- [ ] Remove from TTW Community Calendars
+    - [ ] The Turing Way Community (public-facing)
+    - [ ] The Turing Way Core Team (internal)
+- [ ] Remove from calendar invites for The Turing Way core team
+    - [ ] Coworking Calls
+    - [ ] Collaboration Cafes
+    - [ ] Fireside Chat
+    - [ ] Community Chats
+
+If applicable, see [TPS Project Management Repository](https://github.com/alan-turing-institute/tps-project-management) about being onboarded onto the Tools, Practices, and Systems programme more broadly at the Alan Turing Institute. Other research teams and/or groups may have their own onboarding practices and/or repositories (i.e. the [Research Application Manager (RAM) team](https://github.com/alan-turing-institute/research-application-management) or [Community Management (CM) team](https://github.com/alan-turing-institute/open-community-building)).
+
+## Capturing Institutional Knowledge
+The Research Community Manager will schedule an offboarding call with you 
```

Comment:
I would add a line with the aim of the call, how long it will last and have available a form with some of the questions that this call will be covering.